packages/eslint-config-ibmdotcom/index.js

```diff
@@ -28,26 +32,27 @@ module.exports = {
     'react/no-typos': 2,
     'react-hooks/rules-of-hooks': 'error',
     'react-hooks/exhaustive-deps': 'error',
-    'jsdoc/check-param-names': 2,
-    'jsdoc/check-tag-names': 2,
-    'jsdoc/check-types': 2,
-    'require-jsdoc': [
-      'error',
+    'jsdoc/require-jsdoc': [
+      2,
       {
         require: {
+          ArrowFunctionExpression: true,
+          ClassDeclaration: true,
           FunctionDeclaration: true,
           MethodDefinition: true,
-          ClassDeclaration: true,
-          ArrowFunctionExpression: true,
         },
+        // contexts: ['VariableDeclaration'],
+        exemptEmptyFunctions: false,
```

Comment:
@jeffchew This is causing linting errors and can probably be removed anyway since it defaults to `false`

![image](https://user-images.githubusercontent.com/909118/61304333-fce0fb00-a7b6-11e9-9a97-648ad6e58f5a.png)

Reply:
Sounds good @kennylam , I removed that as well as the commented out entry above.